tests/core/test_proxy.py

```diff
@@ -44,3 +63,50 @@ def test_no_proxy(self):
         env.pop("https_proxy", None)
         env.pop("HTTP_PROXY", None)
         env.pop("HTTPS_PROXY", None)
+
+class TestProxy(TestCase):
+    @attr(requires='test')
```

Comment:
Indeed I forgot to replace that after my testing was done. For the `test_no_proxy` I thought it ran as part of the `default` flavor. Should I add it to the `core_integration` flavor then ?